packages/xo-server-auth-ldap/src/index.js

```diff
@@ -9,7 +10,7 @@ import { readFile } from 'fs'
 
 const DEFAULTS = {
   checkCertificate: true,
-  filter: '(uid={{name}})',
+  filter: '(uid={{name}})'
```

Comment:
Unrelated changes.

Lemme know if I should reformat everything in `master`. :slightly_smiling_face:

Reply:
No, it's fine like that. I fixed it, it was my bad.